tokenauth/auth_backends.py

```diff
@@ -8,11 +8,7 @@
 class EmailTokenBackend:
     def get_user(self, user_id):
         """Get a user by their primary key."""
-        User = get_user_model()
-        try:
-            return User.objects.get(pk=user_id)
-        except User.DoesNotExist:
-            return None
+        return get_user_model().objects.filter(pk=user_id).first()
```

Comment:
I can't believe I had this so verbose. Actually, come to think of it, I think this was from before `.first()`.